.github/workflows/ci.yml

```diff
@@ -93,43 +93,46 @@ jobs:
       - name: Upload coverage reports to Codecov
         uses: codecov/codecov-action@v3
 
-      - name: Stage new files and commit
-        id: stage_files
+      - name: Stage Files and Check for New Cassettes
+        id: stage_files_and_check
         run: |
           git add tests
```

Comment:
Only add the cassets files ending in .yaml similar to your diff command. That should prevent a bunch of issues